src/Microsoft.Health.Fhir.Api/Features/Operations/OperationsCapabilityProvider.cs

```diff
@@ -0,0 +1,118 @@
+            }
+
+            if (_featureConfiguration.SupportsAnonymizedExport)
+            {
+                builder.Update(AddAnonymizedExportDetails);
+            }
+        }
+
+        public void AddExportDetails(ListedCapabilityStatement capabilityStatement)
+        {
+            GetAndAddOperationDefinitionUriToCapabilityStatement(capabilityStatement, OperationsConstants.Export);
+            GetAndAddOperationDefinitionUriToCapabilityStatement(capabilityStatement, OperationsConstants.PatientExport, KnownResourceTypes.Patient);
+            GetAndAddOperationDefinitionUriToCapabilityStatement(capabilityStatement, OperationsConstants.GroupExport, KnownResourceTypes.Group);
+        }
+
+        public void AddAnonymizedExportDetails(ListedCapabilityStatement capabilityStatement)
+        {
+            GetAndAddOperationDefinitionUriToCapabilityStatement(capabilityStatement, OperationsConstants.AnonymizedExport);
+        }
+
+        public void AddReindexDetails(ListedCapabilityStatement capabilityStatement)
+        {
+            GetAndAddOperationDefinitionUriToCapabilityStatement(capabilityStatement, OperationsConstants.Reindex);
+            GetAndAddOperationDefinitionUriToCapabilityStatement(capabilityStatement, OperationsConstants.ResourceReindex);
+        }
+
+        private void GetAndAddOperationDefinitionUriToCapabilityStatement(ListedCapabilityStatement capabilityStatement, string operationType)
+        {
+            Uri operationDefinitionUri = _urlResolver.ResolveOperationDefinitionUrl(operationType);
+            capabilityStatement.Rest.Server().Operation.Add(new OperationComponent()
+            {
+                Name = operationType,
+                Definition = operationDefinitionUri.ToString(),
+            });
+        }
+
+        private void GetAndAddOperationDefinitionUriToCapabilityStatement(ListedCapabilityStatement capabilityStatement, string operationType, string resourceType)
+        {
+            Uri operationDefinitionUri = _urlResolver.ResolveOperationDefinitionUrl(operationType);
+
+            ListedRestComponent listedRestComponent = capabilityStatement.Rest.Server();
+            ListedResourceComponent resourceComponent = listedRestComponent.Resource.SingleOrDefault(x => string.Equals(x.Type, resourceType, StringComparison.OrdinalIgnoreCase));
+
+            if (resourceComponent == null)
+            {
+                resourceComponent = new ListedResourceComponent
+                {
+                    Type = resourceType,
+                    Profile = new ReferenceComponent
+                    {
+                        Reference = $"http://hl7.org/fhir/StructureDefinition/{resourceType}",
+                    },
+                };
+            }
+
+            resourceComponent.Operation.Add(new OperationComponent()
+            {
+                Name = operationType,
+                Definition = operationDefinitionUri.ToString(),
+            });
```

Comment:
Suggestion, if you passed the Builder in here you could use:
```
_builder.UpdateRestResourceComponent(resourceType, resourceComponent => {
resourceComponent.Operation.Add(new OperationComponent()
            {
                Name = operationType,
                Definition = operationDefinitionUri.ToString(),
            });
});
```

It would handle all the other logic in this method

Reply:
Updated code to use the `UpdateRestResourceComponent` method for both `Patient` and `Group` export